kolibri/plugins/facility_management/assets/src/views/class-edit-page/index.vue

```diff
@@ -173,48 +179,24 @@
 
   @require '~kolibri.styles.definitions'
 
-  .toolbar
-    margin-bottom: 32px
-
-  .searchbar
-    margin-top: 5px
-
-  #name-edit-box
+  .title-header
     display: inline-block
-    cursor: pointer
-    margin: 15px 0 5px
-
-  .name-edit
-    float: left
-
-  #edit-name
-    font-size: 1.5em
-    font-weight: bold
 
-  #edit-icon
-    fill: $core-action-normal
-    margin: 2px 0 0 5px
-
-  .toolbar:after
-    content: ''
-    display: table
-    clear: both
-
-  .enroll-user-button
-    width: 100%
-
-  .enroll
-    float: right
+  .edit-button
+    display: inline-block
+    position: relative
+    left: 10px
+    top: -4px
 
-  .empty-list
-    color: $core-text-annotation
-    margin-left: 10px
+  .right
+    text-align: right
 
-  .header
-    font-weight: normal
+  .top-margin
+    margin-top: 24px
 
-  .user-roster
-    overflow-x: auto
-    overflow-y: hidden
+  // overwrite global styling
+  // TODO - find a better way of doing this
```

Comment:
I guess we could give the k-grid a prop that removes padding?

Reply:
really ought to be getting rid of these, and replacing them with something that doesn't introduce left- and right- padding to the outer elements. The goal was just gutters - not outer margins

Reply:
https://github.com/learningequality/kolibri/blob/develop/kolibri/core/assets/src/styles/main.styl#L118